spec/models/user_spec.rb

```diff
@@ -12,6 +12,12 @@
     end
   end
 
+  describe '#become_active' do
+    it 'updates the last_active_at attribute' do
+      expect { user.become_active }.to change { user.last_active_at }
```

Comment:
We should also check here that the `updated_at` column remains the same before and after.

```ruby
it 'does not change the updated_at column' do
  updated_at = user.updated_at
  user.become_active
  expect(updated_at).to eql(user.updated_at)
end
```